RobotArmAPP/Classes/Frames.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.UI.Xaml.Controls;
+
+namespace RobotArmAPP.Classes
+{
+    class Frames
+    {
+        ConvertToString convertToString = new ConvertToString();
+
+        public void InsertFrameFunction(TextBox FrameSpeedBox, TextBox DelayBox, ListView FramesListView, List<int[]> framesList, Slider Eixo1Slider, Slider Eixo2Slider, Slider Eixo3Slider, Slider Eixo4Slider, Slider GarraSlider )
+        {
+            try
+            {
+                int frameSpeed = Convert.ToInt16(FrameSpeedBox.Text);
+                int frameDelay = Convert.ToInt32(DelayBox.Text);
+                string add = convertToString.ConvertItemToString((int)GarraSlider.Value, (int)Eixo4Slider.Value, (int)Eixo3Slider.Value, (int)Eixo2Slider.Value, (int)Eixo1Slider.Value, frameSpeed, frameDelay);
```

Comment:
```suggestion
                string add = movement.ToString();
```
Veja como criar uma classe dedicada removeria este trabalho replicado de copiar valores manualmente.

Reply:
resolvido!